python/ray/serve/_private/logging_utils.py

```diff
@@ -100,7 +100,7 @@ def format(self, record: logging.LogRecord) -> str:
                 The formatted log record in json format.
         """
         record_format = copy.deepcopy(self.component_log_fmt)
-        record_attributes = copy.deepcopy(record.__dict__)
```

Comment:
no idea why this was here in the first place...

Reply:
I feel it's probably a copy pasta from the line above 😅